### ISSUE TITLE
Audit of existing Methodologies

### ISSUE THREAD

**Issue by logicmoo** [fairy-tale-agi-solutions/fairy-tale-artificial-general-intelligence-solutions#13]
Create lists of what has been tried before

With each item, describe the strengths, weakness and blind spots.

**Comment by logicmoo** [comment]
This might be best done on a Wiki?  Initially, these could be pasted from Wikipedia and we'd put our analysis and clarifications?

**Comment by razvan-panda** [comment]
Ideally, we would want one of the best software solutions for doing peer reviews of ideas.
My current intention is to just install Upsource code review in a VM and review ideas which will be committed to the repository.
Natural language is same as code so I don't see why that would not work as well.

**Comment by logicmoo** [comment]
Yes, not every idea has code written yet.  The idea in this issue is to ensuring we do not repeat some existing (failed) paths (initially, because we assumed some methodology has not been tried) This way if we are trying to repeat a model (perhaps the previous inventor made certain errors)  we still salvage the important merits.

**Comment by razvan-panda** [comment]
What I intended for us to do is to put high-level ideas/diagrams in a folder as part of this repository and then we can review them using Upsource.

Whatever hypothesis are promising should be explained as clearly/formally as possible and peer-reviewed for how to improve it.

By high-level I mean just an overview - the entire method can be referenced from wherever it is fully described, maybe someone's personal GitHub repository or some other internet links, etc.

If the high-level ideas are consistent/sound/logical/connecting etc. then it makes more sense to go further into spending time in understanding the other ideas/rest of it.

**Comment by razvan-panda** [comment]
To avoid mistakes I think one of the best ways it to go mainly top-down and make sure that the symbols connecting make sense. That is the reason for the `Defining general intelligence` ticket.

Once you have a high-level logical design then you can keep going lower without worrying as much of making mistakes.

**Comment by razvan-panda** [comment]
@logicmoo I gathered a list here of other AGI projects or organizations: https://github.com/fairy-tale-agi-solutions/awesome-artificial-general-intelligence#organizations--projects

I do not know how we would work on "With each item, describe the methodology, strengths, weaknesses, and blind spots." though, since none of them actually implement an AGI. They just intend to implement an AGI without being successful.
